### DISCUSSION TITLE
Whats up with Jyers??

### DISCUSSION THREAD

**Discussion by PapsDizzy** [Jyers/Marlin#1827] [General Discussion]
Hi @Jyers ,
I hope you are well?! What's the matter with you? you do not report, there are no updates to the firmware from you. I love your firmware but the lack of support made me switch to a different one, which I actually think is a shame. Of course I understand that real life takes precedence and is untouchable. But info for the community on how the firmware will continue or whether it will continue at all would be great. tititopher68-dev tries to keep your firmware up to date and fix bugs, but it's very confusing that he has to post his updates under the discussions. Dear jyers, just get in touch and bring us your community up to date on how things should continue.

**Comment by jetel73** [comment]
+1 some news maybe ?
I'd like titopher68 to be able to give us precompiled versions as you used to do cause a lot of us are here because compiling is not everybody's cup of tea ?
Be safe @+

**Comment by jetel73** [comment]
+1
I'd like titopher68 to have all the abilitations to make the story continue

**Comment by PapsDizzy** [comment]
@Jyers whats up now?? how is it going on with the firmware?

**Reply by K2Kevin** [reply]
The new version V2.0.2i seems pretty perfect for now. I haven't noticed any major bugs anymore.

**Reply by PapsDizzy** [reply]
@K2Kevin 
that is cool but not everybody have the skills and passion for compiling. 
so that @tititopher68-dev  can continue properly, jyers should let us know what's going on or at least  give @tititopher68-dev  the rights to post the versions under the releases.

**Reply by K2Kevin** [reply]
> @K2Kevin
> that is cool but not everybody have the skills and passion for compiling.
> so that @tititopher68-dev can continue properly, jyers should let us know what's going on or at least give @tititopher68-dev the rights to post the versions under the releases.

I could compile for you.  Tell me your specs, and what you want built.

**Reply by K2Kevin** [reply]
The latest Jyers was so full of "ugly" bugs, I'm surprised anyone could get a clean print. So I do understand your pain.

**Reply by PapsDizzy** [reply]
a nice offer, but i switched to another one because of the problems with the previous versions and everything is going well at the moment. if there is another official jyers release I'll be there again. I don't care if it comes from jyers or from tititopher68-dev . I personally would prefer an official release by tititopher68-dev because he's put a lot of work into it lately and jyers doesn't get in touch as if he doesn't care.

**Reply by K2Kevin** [reply]
What is an official release? This is in perpetual beta.
OH, if you found a good alternative, what's the problem.  Seems like Jyers has moved on, taking a well needed break, or just maybe... needs to make a living.  Im glad @tititopher68-dev has taken up the slack and released this version offering some of the new features many of us wanted and squashed a ton of Jyers bugs that really annoyed many of us.
